packages/docs/content/components/button.mdx

```diff
@@ -242,7 +242,7 @@ In addition to the supported props listed, you can also pass in additional
 props, which will be passed to the rendered root component. For example,
 you could pass in a `target` prop to pass to the rendered anchor element.
 
-<PropTable componentName="Button">
+<PropTable componentName="Button" hasAnalytics >
```

Comment:
I found one more annoying exception: Button uses `AnalyticsParentDataProps` as well

Reply:
Ah, I wasn't sure about those, should they go on all that `hasAnalytics` or just button?

Reply:
Nevermind, I see, just for buttons.. updating

Reply:
Updated